ui/guest/src/react/Field.tsx

```diff
@@ -41,10 +42,18 @@ export const Field = forwardRef<any, FieldProps>(function <P = {}>(props: FieldP
     ref
   });
   const Component = component as ComponentType<P>;
+
+  const contentTypeId = model.craftercms.contentTypeId;
+  const contentType = contentController.getCachedContentType(contentTypeId);
+  const field = contentType?.fields[fieldId];
+
   const passDownProps = {
     ...other,
     ...ice,
     ...componentProps,
+    'data-craftercms-model-id': model.craftercms.id,
```

Comment:
Use a different attribute (e.g. `data-craftercms-field`). Value is unnecessary. Add comment stating why we're adding these attributes.